From 5abf9993d76b2e326bf7bac67f152563622be0cc Mon Sep 17 00:00:00 2001
From: lixin <lixin@chengjieos.com>
Date: Tue, 13 Oct 2020 17:37:46 +0800
Subject: [PATCH 2/2] update patch1

---
 hello.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hello.py b/hello.py
index 5aad21d..819889b 100755
--- a/hello.py
+++ b/hello.py
@@ -2,5 +2,4 @@
 
 print('Hello,World')
 print('test')
-print('commit')
-print('patch0')
+print('patch1')
-- 
2.18.2

